builders/integrations/indexers/subquery.md

```diff
@@ -0,0 +1,160 @@
+---
+title: SubQuery
+description: Learn how to use SubQuery to index Substrate and EVM data for Moonbeam and Moonriver
+---
+
+# Indexing Moonbeam with SubQuery
+
+![SubQuery Banner](/images/builders/integrations/indexers/subquery/subquery-banner.png)
+
+## Introduction {: #introduction }
+
+[SubQuery](https://subquery.network/){target=blank} is a data aggregation layer that operates between the layer-1 blockchains (such as Moonbeam and Polkadot) and DApps. This service unlocks blockchain data and transforms it to a queryable state so that it can be used in intuitive applications. It allows DApp developers to focus on their core use case and front end, without needing to waste time on building a custom back end for data processing.
+
+SubQuery supports indexing the Ethereum Virtual Machine (EVM) and Substrate data for any of the Moonbeam networks. A key advantage of using SubQuery is that you can flexibly collect query data across both Moonbeam's EVM and Substrate code with a single project and tool, and then query this data using GraphQL.
+
+For example, SubQuery can filter and query EVM logs and transactions in addition to Substrate data sources. SubQuery introduces more advanced filters than other indexers, allowing filtering of non-contract transactions, transaction senders, contracts and indexed log arguments, so developers can build a wide variety of projects that cater to their specific data needs.
+
+## Getting Started {: #getting-started }
+
+To get started, you'll need to [create a SubQuery project](https://doc.subquery.network/create/introduction/){target=blank}. Once you have your project set up, you can familiarize yourself with [how to connect to and index data from Moonbeam, Moonriver, and Moonbase Alpha](https://doc.subquery.network/create/moonbeam/){target=blank}.
+
+From within your SubQuery project, you can add the custom data source as a dependency by running the following [NPM](https://www.npmjs.com/){target=blank} or [Yarn](https://yarnpkg.com/){target=blank} command:
+
+=== "NPM"
+    ```
+    npm install @subql/contract-processors
+    ```
+
+=== "Yarn"
+    ```
+    yarn add @subql/contract-processors
+    ```
+
+The process to index Moonbeam data takes only two steps: [adding the Moonbeam custom data source](#adding-the-moonbeam-custom-data-source){target=blank} and then [indexing the Moonbeam data](#indexing-moonbeam-data){target=blank}.
+
+## Adding the Moonbeam Custom Data Source {: #adding-the-moonbeam-custom-data-source }
+
+SubQuery has created a data processor specifically made to work with Moonbeam’s implementation of [Frontier](https://github.com/paritytech/frontier){target=blank}. It allows you to reference specific ABI resources used by the processor to parse arguments and the smart contract address that the events are from or the call is made to.
```

Comment:
If I don't know much about Subquery, I have no clue what is a data processor. Maybe add something here about what a data processor is